data/successors/successor hails.txt

```diff
@@ -0,0 +1,393 @@
+# Copyright (c) 2024 by Daeridanii
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Successor civilian
+
+phrase "successor friendly hail" # Wary hails
+	word
+		"Art thou the 'noble visitor' of whom the Houses speak?"
+		"Art thou the 'honored guest' the Houses do parade?"
+		"Art thou the fiendish creature that hails from distant lands?"
+		"Art thou the strange-ish traveler that hails from distant stars?"
+		"Art thou the alien visitor within our noble lands?"
+		"Art thou the creature've history returned to seek the present?"
+		"Thy ship and countenance seem strange-ish and bizarre,"
+	word
+		" / "
+	word
+		"I note thy presence but remain thee wary."
+		"If corrupted be thy soul, then prithee keep thy distance."
+		"If thou art friend to me, so I shall be to thee."
+		"What is thy goal within these stars? To wander in delight?"
+		"To whom and what thou does thou lend thy own strength?"
+		"What is thy goal within these stars? To observe or interfere?"
+		"What is thy goal within these stars? Be a trader or a traitor?"
+
+
+phrase "successor friendly hail" # Brag hails 1
+	word
+		"I never fly without"
+	word
+		" "
+	word
+		"a trusty scanner"
+		"my trusty scanner"
+		"a larger fuel tank"
+		"communications"
+		"some conversation"
+		"a stock of crab-cubes"
+		"a stock of kelp-packs"
+		"a week's supply-cache"
+	word
+		", / "
+	word
+		"I suggest you do the same if you have interest."
+		"It makes the many hours pass by quickly."
+		"The dark of space is lightened by its presence."
+		"There's little point in taking off without it."
+
+phrase "successor friendly hail" # Job hails
+	word
+		"My "
+	phrase
+		"relations"
```

Comment:
Where is the `relations` phrase defined? I can't seem to find it anywhere.

Reply:
It is defined in `successors.txt`.